xbmc/LangInfo.cpp

```diff
@@ -275,6 +275,14 @@ void CLangInfo::CRegion::SetGlobalLocale()
 #endif
   }
 
+  try
+  {
+    g_langInfo.m_originalLocale = std::locale(strLocale.c_str());
+  }
+  catch (...) {
```

Comment:
I'm not sure if there's a chance to get an error there..
I put it in this way because In the same code it's under try-catch the same code:
https://github.com/xbmc/xbmc/blob/master/xbmc/LangInfo.cpp#L298